src/components/productdisplayitem.main.jsx

```diff
@@ -207,6 +221,7 @@ class ProductDisplayItemMain extends React.Component {
           },
           body: JSON.stringify({
             quantity: itemQuantity,
+            configuration: itemConfiguration,
```

Comment:
This will get passed into the request body even if it's empty.

Reply:
Fixed